subprojects/build-cache/src/jmh/java/org/gradle/caching/internal/tasks/AbstractTaskOutputPackagingBenchmark.java

```diff
@@ -44,17 +44,17 @@ public abstract class AbstractTaskOutputPackagingBenchmark {
     private static final DefaultDirectoryProvider DIRECTORY_PROVIDER = new DefaultDirectoryProvider();
 
     private static final Map<String, Packer> PACKERS = ImmutableMap.<String, Packer>builder()
-        .put("tar.snappy", new SnappyPacker(new AntTarPacker(4)))
-        .put("tar.snappy.commons", new SnappyCommonsPacker(new AntTarPacker(4)))
-        .put("tar.snappy.dain", new SnappyDainPacker(new AntTarPacker(4)))
-        .put("tar.snappy.small", new SnappyPacker(new AntTarPacker(2)))
-        .put("tar.snappy.large", new SnappyPacker(new AntTarPacker(64)))
-        .put("tar", new AntTarPacker(4))
+        .put("tar.snappy", new SnappyPacker(new CommonsTarPacker(4)))
+        .put("tar.snappy.commons", new SnappyCommonsPacker(new CommonsTarPacker(4)))
+        .put("tar.snappy.dain", new SnappyDainPacker(new CommonsTarPacker(4)))
+        .put("tar.snappy.small", new SnappyPacker(new CommonsTarPacker(2)))
+        .put("tar.snappy.large", new SnappyPacker(new CommonsTarPacker(64)))
+        .put("tar", new CommonsTarPacker(4))
         .put("tar.commons", new CommonsTarPacker(4))
         .put("tar.jtar", new JTarPacker(4))
-        .put("tar.small", new AntTarPacker(2))
-        .put("tar.large", new AntTarPacker(64))
-        .put("tar.gz", new GzipPacker(new AntTarPacker(4)))
+        .put("tar.small", new CommonsTarPacker(2))
+        .put("tar.large", new CommonsTarPacker(64))
+        .put("tar.gz", new GzipPacker(new CommonsTarPacker(4)))
```

Comment:
@lptr this replaces the Ant TAR packer with the Apache Commons Compress one. Are you ok with this change? Note that this PR also simply deletes `AntTarPacker`.

Reply:
Sure. But I think we should just remove these benchmarks. It's not like we are using them at all.

Reply:
Thanks for the feedback. We'll leave removing the benchmarks to you.